acceptance/lib/facter/acceptance/base_fact_utils.rb

```diff
@@ -492,7 +492,7 @@ def ubuntu_expected_facts(agent)
         expected_facts = {
             'os.architecture'          => os_arch,
             'os.distro.codename'       => /\w+/,
-            'os.distro.description'    => /Ubuntu #{os_version}/,
+            'os.distro.description'    => agent['platform'] == 'ubuntu-20.04-amd64' ? 'Ubuntu Focal Fossa (development branch)' : /Ubuntu #{os_version}/,
```

Comment:
I think something similar happened with Debian 10 beta, but it was changed at release.